template.html

```diff
@@ -378,19 +384,42 @@ <h3>{{topic.others.by-us-title}}</h3>
     </footer>
 
     <script>
-      // ⚠ Contents of this script block are hashed and used in
-      // the CSP meta tag. Any change (including white space) will
-      // require update of the CSP tag. All this to prevent the
-      // need for 'unsafe-inline'.
+      // ⚠ The contents of this script block are hashed and used in the CSP meta tag. 
+      // Any change (including white space) will require an update of the CSP tag. 
+      // All this to prevent the need for 'unsafe-inline'.
+
       document.addEventListener("DOMContentLoaded", () => {
-        [...document.querySelectorAll(".details-toggle")].forEach(element => {
-          const anchor = element.dataset.toggleTarget;
-          element.addEventListener('click', () => {
-            document.getElementById('viewToggle').classList.toggle('view-overview');
-            location.hash = anchor;
-          });
-        });
+
+        const url = new URL(window.location.href);
+        const button = document.querySelector('[data-toggle="summary"]');
+
+        if (url.searchParams.get('summary') === 'true') {
+          document.getElementById('viewToggle').classList.toggle('view-summary');
+          button.setAttribute('aria-checked', 'false');
+          button.setAttribute('aria-expanded', 'false');
+        }
+
+        button.addEventListener('click', toggleDetailView);
       });
+
+      function toggleDetailView() {
+        const viewToggleElem = document.getElementById('viewToggle');
```

Comment:
Instead of using an `id` for the `body`, you could consider using `document.body` directly. That might make the code slightly more clear.

Reply:
Fair, fair.